furusiya/model/components/xp.py

```diff
@@ -0,0 +1,27 @@
+from main_interface import message
+from model.components.base import Component
+from model.config import config
+from model.systems.fighter_system import FighterSystem
+
+
+class XP(Component):
```

Comment:
This should really be called `XPComponent`, I know it's redundant though. Clarity! Reading random code, I might think this represents a bunch of experience points...